### PR TITLE
Add build for Windows and create postinstall script that is cross-platform

### LINKED ISSUE #13: Add support for Windows
As far as I understand the configured workflow, it does not build a binary for Windows. We do have a team member with a Windows laptop thus we need to support that platform. I'm happy to help set this up (although I'm very new to the Github Actions) or test. Thanks!

### PR DESCRIPTION
**Pull request by romanschejbal**
# Fixes #13 
I have:
1) split build into respective steps, windows doesn't like the forward-slash in paths
1) added windows support
1) converted the post-install script into node as it needs to be cross-platform from now on
1) prettier-formatted the pipeline.yml
1) added menhir as dependency + resolution since esy fails to build on windows otherwise, see this https://github.com/esy/esy/issues/913 for more deets

If you guys could look at it and review that would be great! :) 

Thanks

### CODE REVIEW

**Comment by fakenickels** [comment]
Thanks a lot for this! We'll be taking a look and merging it soon!

File: packages/ppx/postinstall.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const { exec } = require("child_process");
+const fs = require("fs");
+
+exec("uname", (error, stdout, stderr) => {
```
Comment (fakenickels) [inline packages/ppx/postinstall.js]: now that we are using a node script for this we can just use `process.platform` instead of uname!

Comment (romanschejbal) [reply packages/ppx/postinstall.js]: Done :-) thanks

**Comment by fakenickels** [comment]
Thanks a lot! Will publish soon

**Pull request merged**